core/Updates/5.0.0-b1.php

```diff
@@ -48,6 +51,14 @@ public function getMigrations(Updater $updater)
         $migrations = $this->getUpdateArchiveIndexMigrations();
 
         $migrations[] = $this->migration->db->addColumns('user_token_auth', ['post_only' => "TINYINT(2) UNSIGNED NOT NULL DEFAULT '0'"]);
+        $migrations[] = $this->migration->db->addColumns('log_conversion', ['pageviews_before' => "SMALLINT UNSIGNED DEFAULT NULL"]);
+
+        $instanceId = SettingsPiwik::getPiwikInstanceId();
+        if (strpos($instanceId, '.matomo.cloud') === false || strpos($instanceId, '.innocraft.cloud') === false) {
```

Comment:
That is currently actually always `true`
```suggestion
        if (strpos($instanceId, '.matomo.cloud') === false && strpos($instanceId, '.innocraft.cloud') === false) {
```

Reply:
:facepalm: